src/mrinufft/extras/data.py

```diff
@@ -1,106 +1,75 @@
-"""Field map generator module."""
+"""Data generator module."""
 
 import numpy as np
 
 
-def make_b0map(shape, b0range=(-300, 300), mask=None):
-    """
-    Make radial B0 map.
-
-    Parameters
-    ----------
-    shape : tuple[int]
-        Matrix size. Only supports isotropic matrices.
-    b0range : tuple[float], optional
-        Frequency shift range in [Hz]. The default is (-300, 300).
-    mask : np.ndarray
-        Spatial support of the objec. If not provided,
-        build a radial mask with radius = 0.3 * shape
-
-    Returns
-    -------
-    np.ndarray
-        B0 map of shape (*shape) in [Hz],
-        with values included in (*b0range).
-    mask : np.ndarray, optional
-        Spatial support binary mask.
-
-    """
-    assert np.unique(shape).size, ValueError("Only isotropic matriex are supported.")
-    ndim = len(shape)
-    if ndim == 2:
-        radial_mask, fieldmap = _make_disk(shape)
-    elif ndim == 3:
-        radial_mask, fieldmap = _make_sphere(shape)
-    if mask is None:
-        mask = radial_mask
-
-    # build map
-    fieldmap *= mask
-    fieldmap = (b0range[1] - b0range[0]) * fieldmap / fieldmap.max() + b0range[0]  # Hz
-    fieldmap *= mask
-
-    # remove nan
-    fieldmap = np.nan_to_num(fieldmap, neginf=0.0, posinf=0.0)
+def get_brainweb_map(sub_id: int) -> np.ndarray:
+    """Get MRI parametric maps from a brainweb crisp segmentation.
```

Comment:
by MRI do you refer to relaxation (T1, T2) maps?

Reply:
Yes, I have specified that now.